javascript/SubsitesTreeDropdownField.js

```diff
@@ -8,7 +8,8 @@
 			onchange: function() {
 				// TODO Data binding between two fields
                 const name = this.attr('name').replace('_SubsiteID', '');
-                let field = $('#Form_EditForm_' + name).first();
+                const formId = this.closest('form').attr('id');
+                let field = $('#' + formId + '_' + name).first();
```

Comment:
I think that `const` is OK here, as the identifier `field` is never reassigned :)
as I understand it, mutation is OK, reassignment is not.
e.g. `const arr = []; arr.push('anything');` is fine.